### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by maximuspc** [Openarl/PathOfBuilding#1268]
As a POE newbie, I want to see which level a node was taken at in the progression of a build as this gives me a rough idea of the build order for nodes and avoids costly mistakes.

**Comment by niuage** [comment]
I don't think that would work well. PoB can't guess the order, so the build creator would have to "record" himself selecting the points for the whole tree, and that's getting complicated. That would be even worse for scion, which usually has to respec points because they can start from different classes later on.

But PoB allows any given build to have multiple trees (you can switch between them on the bottom left corner of the tree), so when people are making PoBs for beginners, they usually include many leveling trees (one at 20 points, one at 40 etc). That should be enough to tell you how to level.
